apps/prairielearn/src/components/SyncErrorsAndWarnings.html.tsx

```diff
@@ -0,0 +1,186 @@
+import clsx from 'clsx';
+
+import { ansiToHtml } from '../lib/chalk.js';
+import type { StaffCourse, StaffCourseInstance } from '../lib/client/safe-db-types.js';
+import { type Assessment, type Question } from '../lib/db-types.js';
+
+export function CourseSyncErrorsAndWarnings({
+  authz_data,
+  course,
+  urlPrefix,
+}: {
+  authz_data: { has_course_instance_permission_edit: boolean };
+  course: StaffCourse;
+  urlPrefix: string;
+}) {
+  return (
+    <SyncErrorsAndWarnings
+      authz_data={authz_data}
+      exampleCourse={course.example_course}
+      syncErrors={course.sync_errors}
+      syncWarnings={course.sync_warnings}
+      fileEditUrl={`${urlPrefix}/course_admin/file_edit/infoCourse.json`}
+      context={'course'}
```

Comment:
Nit, I typically prefer this style for string props. Ditto elsewhere.

```suggestion
      context="course"
```

Reply:
Changed in d46cc20f82875c50de899e50dd45e4fc85c4616c.